### ISSUE TITLE
Merge Config

### ISSUE THREAD

**Issue by allo-** [allo-/firefox-profilemaker#25]
A (maybe javascript only) function to merge an existing prefs.js with the generated options.

**Comment by StarPicard** [comment]
Yes, with #107 i meant something like this function.

**Comment by wefalltomorrow** [comment]
Has this been achieved yet? Would be nice to keep some of my configs whilst also utilizing some of the configs this neat little tool offers.

**Comment by allo-** [comment]
No. And I wonder if it should be combined with providing own profiles (Sets of questions), which should allow profiles that are not the main focus to be provided by others (or inofficially provided by the project) like UI customizations.

**Comment by wefalltomorrow** [comment]
That would definitely be neat!
